upgrade_lalouve.py

```diff
@@ -0,0 +1,117 @@
+# -*- coding: utf-8 -*-
+from openerp.tools import config
+
+
+def run(session, logger):
+    """Update all modules."""
+    if session.is_initialization:
+        config['load_language'] = 'fr_FR'
+        modules = [
+#   'coop_product_coefficient',
+# 'pos_product_barcodes',
+# 'pos_automatic_cashdrawer',
+
+        'coop_membership_access_rights',
+        'smile_base',
+        'louve_custom_product',
+        'louve_custom_purchase',
+        'account_payment_transfer_account',
+        'account_asset',
+        'coop_purchase',
+        'louve_custom_account',
+        'louve_custom_pos',
+        'portal_sale',
+        'account_product_fiscal_classification',
+        'board',
+        'l10n_fr_fec',
+        'louve_custom_inventory',
+        'louve_welcome_email',
+        'mail_tip',
+        'partner_validate_email',
+        'portal_stock',
+        'pos_payment_terminal_return',
+        
+        'product_history_for_cpo',
+        'stock_expense_transfer',
+        # en dessous de la ligne install 1 dépendance
+        'account_bank_statement_import_caisse_epargne',
+'account_bank_statement_import_ofx',
```

Comment:
Could you use the same indent, flake8 would complains about that! (it doesn't because we are not testing this file: https://travis-ci.org/dlemireanybox/odoo-foodcoops/builds/289931019#L4293)